docs/conf.py

```diff
@@ -0,0 +1,83 @@
+# Configuration file for the Sphinx documentation builder.                                                                                                                 
+#                                                                                                                                                                          
+# This file only contains a selection of the most common options. For a full                                                                                               
+# list see the documentation:                                                                                                                                              
+# http://www.sphinx-doc.org/en/master/config                                                                                                                               
+
+# -- Path setup --------------------------------------------------------------                                                                                             
+
+# If extensions (or modules to document with autodoc) are in another directory,                                                                                            
+# add these directories to sys.path here. If the directory is relative to the                                                                                              
+# documentation root, use os.path.abspath to make it absolute, like shown here.                                                                                            
+#                                                                                                                                                                          
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
+print(sys.path)
+
+# -- Project information -----------------------------------------------------                                                                                             
+
+project = 'United Workflow'
+author = 'NOAA/EMC, NOAA/GSL, GMU, CIRES, and UCAR/NCAR'
+author_list = 'Mahajan, R., C. Holt, B. Cash, F. Gabelmann, O. Adejumo, and J. Prestopnik'
+version = '0.0.0'
+verinfo = version
+release = f'{version}'
+release_year = '2022'
+release_date = f'{release_year}-03-31'
```

Comment:
Can we have a single `release_date`
e.g.
```
release_date = '2022-03-31'
```
and deduce `release_year`, `release_month`, and `release_day` from it?

Reply:
Yes, and I just modified the file to do that.